andela/lotus-ah
# #161669694 refactor send email verification process, to use dynamic URL

#### What does this PR do?
modify confirm email URL sent to a user upon registration to redirect to a client app

#### Description of Task to be completed?
- add client link as redirect URL on email verification
- modify email links, on user registration email and password reset email
- return user object after updating user

#### How should this be manually tested?
- Setup a client app
- make a request to ```POST /api/v1/users``` passing the email on the request body
- Go to your email to confirm 
- the confirmation link should be in this format
   ```BASE_URL/verifyEmail?token=token```

#### Any background context you want to provide?
The was a challenge redirection to a client after sending email verification because of the 
confirmation URL on the email redirects to the server, hence the need for this.

#### What are the relevant pivotal tracker stories?
[#161669694](https://www.pivotaltracker.com/story/show/161669694, "refactor email verification to redirect to client")

#### Screenshots (if appropriate)
<img width="1396" alt="screen shot 2018-11-02 at 7 30 41 am" src="https://user-images.githubusercontent.com/24890605/47897307-80e59380-de71-11e8-810b-c593aea8ca6b.png">

#### Questions:
NA
